components/esp_hw_support/port/esp32c6/cpu_region_protect.c

```diff
@@ -128,7 +128,7 @@ void esp_cpu_configure_region_protection(void)
 
     if (esp_cpu_dbgr_is_attached()) {
         // Anti-FI check that cpu is really in ocd mode
-        ESP_FAULT_ASSERT(esp_cpu_dbgr_is_attached());
+//        ESP_FAULT_ASSERT(esp_cpu_dbgr_is_attached());
```

Comment:
What is the reason to leave this out?

Reply:
It is part of the original C6 MR. We can uncomment and test it.